package.json

```diff
@@ -30,8 +30,9 @@
   },
   "devDependencies": {
     "@alaskaairux/design-tokens": "^3.13.0",
-    "@aurodesignsystem/webcorestylesheets": "^4.2.0",
+    "@aurodesignsystem/design-tokens": "^4.1.1",
```

Comment:
Remove the legacy `@alaskaairux/design-tokens` and also update the `peerDependencies` to reference the newer `@aurodesignsystem/design-tokens`

Reply:
sure

Reply:
After recent discussions on the Auro team we need to keep both for the time being per earlier comment.